ansible/roles/openstack-ansible-os_triliovault_wlm/tasks/install-ubuntu.yml

```diff
@@ -186,5 +186,15 @@
     - "'{{ physical_host}}' == '{{ WLM_CRON_CONTROLLER_HOSTNAME }}'"
     - WLM_CRON_CONTROLLER_HOSTNAME != ""
 
-
-
+- name: Disable wlm-cron service except {{WLM_CRON_CONTROLLER_HOSTNAME}} host
+  service:
+    name: "{{ item }}"
+    state: stopped
+    enabled: no
+  with_items:
+    - wlm-cron
+  tags:
+    - update_cfg
+  when:
+    - "'{{ physical_host}}' != '{{ WLM_CRON_CONTROLLER_HOSTNAME }}'"
```

Comment:
There is another variable available for hostname, which matches with inventory hostname. Please use that instead of physical_host.